masschallenge/opsworks-web-python
# AC-6694: Create two AWS test stacks for builds

#### Changes introduced in [AC-6694](https://masschallenge.atlassian.net/browse/AC-6694)
- add logic that separates python <= 2.7 and python > 3.4 to allow our opsworks scripts work in both python 2.7 & 3.6

#### How to test
- see the stacks on [AWS](https://console.aws.amazon.com/opsworks/home?region=us-east-1), the stacks are now up and will be accessed on
ac6694.masschallenge.org and
ac6694-2.masschallenge.org
once python compatibility is handled

More details on the [ticket](https://masschallenge.atlassian.net/browse/AC-6694?focusedCommentId=40722&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-40722)
